interface-definitions/system-keyboard.xml.in

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="system">
+    <children>
+      <leafNode name="keyboard-type" owner="${vyos_conf_scripts_dir}/system-keyboard.py">
+        <properties>
+          <help>System keyboard type</help>
+          <completionHelp>
+            <list>azerty</list>
+          </completionHelp>
+          <valueHelp>
+            <format>azerty</format>
+            <description>Set azerty as default keyboard layout</description>
+          </valueHelp>
+          <constraint>
+            <regex>^azerty$</regex>
+          </constraint>
+        </properties>
+      </leafNode>
+    </children>
+  </node>
```

Comment:
Users may misunderstand the meaning of this command, I do not agree to use a seemingly universal command to set the only layout, which will mislead vyos users

Reply:
Any logical proposition?

Reply:
I have made it clear in the task list of T3038 that this will mislead users greatly. I am against the misleading design of CLI